bzt/swagger2yaml.py

```diff
@@ -336,11 +335,12 @@ def _extract_scenarios_from_paths(self, paths):
 
         return scenarios
 
-    def convert(self, swagger_path):
-        if not os.path.exists(swagger_path):
-            raise ValueError("Swagger file %s doesn't exist" % swagger_path)
+    def convert_path(self, swagger_path):
```

Comment:
You ate the friendly error message, it will fail with ugly "IOError" instead.

Reply:
IOError will be captured in `parse()` method and will get rethrown as TaurusConfigError, so it's fine.

Also, `convert_path` should be checking for file existence before opening it.